File: 4. 프로그래밍 언어활용/source/day07/src/exam01/Ex01.java
package exam01;

public class Ex01 {
    public static void main(String[] args) {
        Outer outer = new Outer(); // Outer 클래스를 객체로 생성
        Calculator cal2 = outer.method(30);
        int result = cal2.add(10,20);
        System.out.println(result);
    }
}

File: 17. Python 기초 및 응용/Fileex/ex01.py
class Calculator:
    def setdata(self, a, b):
        self.a = a
        self.b = b

cal1 = Calculator()
cal1.setdata(10, 20)
print(cal1.a, cal1.b)

cal2 = Calculator()
Calculator.setdata(cal2, 20, 30)
print(cal2.a, cal2.b)

cal3 = Calculator()
print(id(cal1), id(cal3))
